lighthouse-extension/package.json

```diff
@@ -23,5 +23,8 @@
     "gulp-zip": "^3.2.0",
     "run-sequence": "^1.1.5",
     "through2": "^2.0.1"
+  },
+  "dependencies": {
+    "sourcemapped-stacktrace": "^1.1.3"
```

Comment:
i think in practical terms it doesnt matter, but i agree that semantically it fits in deps more than devDeps